library/src/test/java/org/apache/apex/malhar/lib/fs/s3output/S3InitiateUploadTest.java

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
The tests seem to be excluded from the build.
Can you try looking for some embedded S3 server for testing? Or write mock tests which simulate the behavior of S3 in order to have the unit testing?